Client/playlist.cpp

```diff
@@ -51,6 +51,7 @@ QIODevice* UrlDispatcher::openUrl(QUrl url)
 
 	QNetworkRequest request;
 	request.setUrl(url);
+	request.setSslConfiguration(QSslConfiguration::defaultConfiguration());
```

Comment:
Is this really necessary? It seems to break the build on Windows, but documentation suggests this is already the default SSL config.

Reply:
Not likely, I think I have it in Audiotest because on my Unix host I wasn't able to connect to HTTPS hosts without it. Removed it for now as we do not even have streamed webaudio support in the game itself yet. We seem to download the whole track before using it.